content/90.back-matter.md

```diff
@@ -3,56 +3,133 @@
 <!-- Explicitly insert bibliography here -->
 <div id="refs"></div>
 
+## Supplemental Section
+
+### Document embeddings derived from bioRxiv reveal fields and subfields
+
+![
+**A.** Principal components (PC) analysis of bioRxiv word2vec embeddings groups documents based on author-selected categories.
+We visualized documents from key categories on a scatterplot for the first two PCs.
+The first PC separated cell biology from informatics-related fields, and the second PC separated bioinformatics from neuroscience fields.
+**B.** A word cloud visualization of PC1.
+Each word cloud depicts the cosine similarity score between tokens and the first PC.
+Tokens in orange were most similar to the PC's positive direction, while tokens in blue were most similar to the PC's negative direction.
+The size of each token indicates the magnitude of the similarity.
+**C.** A word cloud visualization of PC2, which separated bioinformatics from neuroscience.
+Similar to the first PC, tokens in orange were most similar to the PC's positive direction, while tokens in blue were most similar to the PC's negative direction.
+The size of each token indicates the magnitude of the similarity.
+**D.** Examining PC1 values for each article by category created a continuum from informatics-related fields on the top through cell biology on the bottom.
+Specific article categories (neuroscience, genetics) were spread throughout PC1 values.
+**E.** Examining PC2 values for each article by category revealed fields like genomics, bioinformatics, and genetics on the top and neuroscience and behavior on the bottom.
+](https://raw.githubusercontent.com/danich1/annorxiver/5a593f0250886192650dd329e6968069b13e4fcb/figure_generation/output/figure_two_panels.png){#fig:topic_analysis_panels width="100%"}
+
+Document embeddings provide a means to categorize the language of documents in a way that takes into account the similarities between terms [@arxiv:1405.4053; @arxiv:1707.02377; @arxiv:2001.05727].
+We found that the first two PCs separated articles from different author-selected categories (Figure {@fig:topic_analysis_panels}A).
+Certain neuroscience papers appeared to be more associated with the cellular biology direction of PC1, while others seemed to be more associated with the informatics-related direction Figure {@fig:topic_analysis_panels}A).
```

Comment:
If this is in the supplement, I think these also have to be supplementary figures. Can you remind me - we moved this to address a reviewer concern that the paper was too long and complicated?

Reply:
>I would cut this entire subsection from the manuscript to make it shorter (or relegate it to a supplementary section).
>
>Don't we already know that if one uses full texts we can determine the subfield of the paper? It's not that interesting in my opinion and not relevant to the main hypotheses of the paper -- comparing between preprints and the journal published version.

The reviewer found that whole section to be not interesting, so they suggested we remove this section or push it to the supplement. I think removing is extreme and supplement would suffice if it has to be changed.

Reply:
Ok - then I think you definitely need to have a way for folks to find this, so it needs some brief mention of the results and the existence of the analysis in the supplement with a reference to (Supplementary Results).

Reply:
And all of these figures then need to be referred to / numbered as supplementary figs.

Reply:
Ahh - I see the new sentence now!